Yubico.YubiKey/src/Yubico/YubiKey/Cryptography/RsaFormat.cs

```diff
@@ -79,6 +79,23 @@ public static class RsaFormat
         /// </summary>
         public const int KeySizeBits2048 = 2048;
 
+        /// <summary>
+        /// Use this value to indicate the key size, in bits, is 3072. The
+        /// <c>KeySizeBits</c> values listed in this class are the sizes
+        /// supported and provided as a convenience to the user to verify the
+        /// supported sizes.
```

Comment:
This is confusing. When you say that they are "provided as a convenience to the user to verify the supported sizes", how would a user do this in practice?

Reply:
I followed a long in the pattern the devs had established. However, I'm thinking that it might be a better idea to not include this at all. Instead, in a later version of the SDK, include a proper datastructure for PivAlgorith which has size, type, tag, etc. That way the code would be cleaner and people wouldn't have to go looking for constants like these all over the place. What do @AdamVe @jensutbult think?

Reply:
We decided to keep it as it is for now.. But yes, we agree. The intent of these constants are not clear. I suspect that they might have been intended to use as input parameters to some of the `RsaFormat` class methods